src/main/java/nl/novi/techiteasydto/services/TelevisionService.java

```diff
@@ -0,0 +1,68 @@
+package nl.novi.techiteasydto.services;
+
+import nl.novi.techiteasydto.mappers.TelevisionDtoMapper;
+import nl.novi.techiteasydto.models.ResponseTelevisionDto;
+import nl.novi.techiteasydto.models.Television;
+import nl.novi.techiteasydto.repositories.TelevisionRepository;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class TelevisionService {
+    private final TelevisionRepository televisionRepository;
+
+    public TelevisionService(TelevisionRepository televisionRepository, TelevisionDtoMapper toDto) {
+        this.televisionRepository = televisionRepository;
+        this.toDto = toDto;
```

Comment:
Je noemt de mapper hier "toDto". Dat is geen goede naam voor een TelevisionMapper